x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_about_rule/index.test.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { ThemeProvider } from 'styled-components';
+import euiDarkVars from '@elastic/eui/dist/eui_theme_light.json';
+
+import { StepAboutRule } from './';
+import { mockAboutStepRule } from '../../all/__mocks__/mock';
+import { StepRuleDescription } from '../description_step';
+import { stepAboutDefaultValue } from './default_value';
+
+const theme = () => ({ eui: euiDarkVars, darkMode: true });
+
+describe('StepAboutRuleComponent', () => {
+  test('it renders StepRuleDescription if isReadOnlyView is true and `name` property exists', () => {
+    // see mockAboutStepRule for name property
+    // Note: is the name check for old rules? It's required so no rules should ever not include it
```

Comment:
Is this question for someone on the team? Do you want to keep this question here in the code.

Reply:
They were meant to remind me of things to follow up with Xavier on. Went ahead and removed just so I don't mistakenly forget to remove them beforehand.